samples/excel/90-just-for-fun/color-wheel.yaml

```diff
@@ -0,0 +1,144 @@
+id: excel-90-just-for-fun-color-wheel-
+name: Wheel of colors
+description: Uses chart formatting to draw a wheel with changing colors. Contributed by Alexander Zlatkovski.
+author: AlexanderZlatkovski
+host: EXCEL
+api_set:
+    ExcelApi: 1.1
+script:
+    content: |
+        $("#wheel").click(wheelGo);
+        $("#stop").click(wheelStop);
+
+        let stop: boolean
+
+        const pauseLength = 50;
+        const numberOfSlices = 32;
+
+        async function wheelGo() {
+            try {
+                Excel.run(async (context) => {
+                    stop = false
+
+                    // Create a hidden sheet which will contain data for the color wheel chart:
+                    const sheetSettings = await OfficeHelpers.ExcelUtilities.forceCreateSheet(context.workbook, "Color Wheel Settings", true /*clearOnly*/);
+                    sheetSettings.visibility = Excel.SheetVisibility.hidden;
+                    const dataRange = sheetSettings.getRange("A1:A" + numberOfSlices);
+                    const matrix = new Array();
+                    for (let r = 0; r < numberOfSlices; r++) {
+                        matrix[r] = new Array();
+                        for (let c = 0; c < 1; c++) {
+                            matrix[r][c] = 1;
+                        }
+                    }
+                    dataRange.values = matrix;
+
+                    // Create a sheet for the color wheel and format it
+                    const sheet = await OfficeHelpers.ExcelUtilities.forceCreateSheet(context.workbook, "Color Wheel", true /*clearOnly*/);
+                    
+                    sheet.charts.load("id");
+                    sheet.activate();
```

Comment:
Creating a new sheet and activating it with the same context.sync() will cause an issue in Excel Online. Insert a context.sync() right before sheet.activate().